ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -26,7 +26,7 @@
 {%             endif %}
 | {{ ethernet_interface }} | {{ description }} | *{{ mode }} | *{{ vlans }} | *{{ native_vlan }} | *{{ l2.trunk_groups }} | {{ channel_group }} |
 {%         endif %}
-{%     elif ethernet_interfaces[ethernet_interface].type is not defined or ethernet_interfaces[ethernet_interface].type != "routed" %}
+{%     elif ethernet_interfaces[ethernet_interface].type is not defined or ethernet_interfaces[ethernet_interface].type in ["switched", "l2dot1q"]  %}
```

Comment:
Not sure why, but this change removes `switchport` from many of the molecule configs. I don't think that is on purpose.

Reply:
Ok I am looking at why. But as it is on documentation template, I don't think it is the root cause of the change you saw in CLI output.

probaably comes from this line instead: https://github.com/titom73/ansible-avd/blob/6c201e7df7a6e5db599fb3e60a8b9a7d354873da/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2#L35

I considered type would be always defined but I will add a default statement to fallback to interface_defaults or `switched`

Reply:
Fixed with commit 988df1e and artifacts updated. Let me know your thoughts about update